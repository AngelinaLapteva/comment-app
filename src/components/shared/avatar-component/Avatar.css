.Avatar {
    width: auto;
    height: 40px;
    clip-path: circle(20px at center);
    margin-right: 15px;
  }