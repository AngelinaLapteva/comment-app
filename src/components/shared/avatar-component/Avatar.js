import React from 'react';
import './Avatar.css'

const avatar = props => {
    return (
        <img className="Avatar" alt="profile" src={props.avatarPath} />
    )
}
export default avatar;