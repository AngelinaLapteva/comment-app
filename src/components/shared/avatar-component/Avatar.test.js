import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from './Avatar'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })
describe(' Avatar componrnt', () => {
    test('creating component', () => {
        const avWrap = shallow(<Avatar />)
        expect(avWrap.find('img')).toHaveLength(1);
    })
    test('tests avatar component snapshot', () => {
        const component = renderer.create(
            <Avatar avatarPath='/img/catAvatr.png' />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})
