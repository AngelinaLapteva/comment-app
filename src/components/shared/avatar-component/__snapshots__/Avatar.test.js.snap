// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Avatar componrnt tests avatar component snapshot 1`] = `
<img
  alt="profile"
  className="Avatar"
  src="/img/catAvatr.png"
/>
`;
